### PR TITLE
Use Symfony string for most string operations

### PR DESCRIPTION
**Pull request by javiereguiluz**
I think the improved readability is better than the slight performance decrease ... but I'm open to read different opinions. Thanks!

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: src/CI/MissingFilesChecker.php [hunk 1]
```diff
@@ -32,13 +33,8 @@ public function getMissingFiles(): array
...
 
         foreach ($finder as $file) {
-            $sourcePath = ltrim(substr($file->getPathname(), \strlen($this->buildContext->getSourceDir())), '/');
-
-            $htmlFile = sprintf(
-                '%s/%s.html',
-                $this->buildContext->getOutputDir(),
-                substr($sourcePath, 0, -4)
-            );
+            $sourcePath = u($file->getPathname())->after($this->buildContext->getSourceDir())->trimEnd('/');
```
Comment (nikophil) [inline src/CI/MissingFilesChecker.php:36]: because we're doing `ltrim` shouldn't we use `trimStart`?

**Review by nikophil — COMMENTED** [review]
i've not used that much string component so far, readability is really nice :+1:

File: src/CI/MissingFilesChecker.php (hunk 1 above)
Comment (javiereguiluz) [reply src/CI/MissingFilesChecker.php]: You are right! Fixed. Thanks!

**Comment by weaverryan** [comment]
Thank you!

**Pull request merged**